### ISSUE TITLE
Improve accessibility of the header

### ISSUE THREAD

**Issue by Manvel** [browser-automation/cba-website#17]
# Background

With #5 we have implemented header according to the Drupal website, but the problem is that Drupal version of https://chrome-automation.com/ header isn't accessible. ex.: Missing keyboard Navigation and we barely using aria attributes, in order to improve accessibility best would be to follow w3c website recommendation about implementing Fly-out menus.

# What to change

Improve accessibility of the existing header menu using best practices, suggested by w3c.

References: 
- https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html
- https://www.w3.org/TR/wai-aria-practices/#Listbox
